packages/vector_graphics_compiler/test/parser_test.dart

```diff
@@ -704,6 +704,85 @@ void main() {
     ]);
   });
 
+  test('stroke-width with invalid value', () {
+    const String svg =
+        '<svg viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg"><path d="M100 10 H180 V90 H100 Z" fill="#ff0000" stroke="#0000ff" stroke-width="invalid"/></svg>';
+
+    final VectorInstructions instructions = parseWithoutOptimizers(svg);
+
+    expect(instructions.paints, const <Paint>[
+      Paint(
+          stroke: Stroke(color: Color(0xff0000ff)),
+          fill: Fill(color: Color(0xffff0000))),
+    ]);
+
+    expect(instructions.paths, <Path>[
+      Path(
+        commands: const <PathCommand>[
+          MoveToCommand(100.0, 10.0),
+          LineToCommand(180.0, 10.0),
+          LineToCommand(180.0, 90.0),
+          LineToCommand(100.0, 90.0),
+          CloseCommand(),
+        ],
+      ),
+    ]);
+  });
+
+  test('stroke-width with unit value', () {
+    const SvgTheme theme = SvgTheme();
+
+    const String svg_px =
+        '<svg viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg"><path d="M100 10 H180 V90 H100 Z" fill="#ff0000" stroke="#0000ff" stroke-width="1px"/></svg>';
+    const String svg_pt =
+        '<svg viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg"><path d="M100 10 H180 V90 H100 Z" fill="#ff0000" stroke="#0000ff" stroke-width="1pt"/></svg>';
+    const String svg_ex =
+        '<svg viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg"><path d="M100 10 H180 V90 H100 Z" fill="#ff0000" stroke="#0000ff" stroke-width="1ex"/></svg>';
+    const String svg_em =
+        '<svg viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg"><path d="M100 10 H180 V90 H100 Z" fill="#ff0000" stroke="#0000ff" stroke-width="1em"/></svg>';
+    const String svg_rem =
+        '<svg viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg"><path d="M100 10 H180 V90 H100 Z" fill="#ff0000" stroke="#0000ff" stroke-width="1rem"/></svg>';
+
+    final VectorInstructions instructionsPx = parseWithoutOptimizers(svg_px);
+    final VectorInstructions instructionsPt = parseWithoutOptimizers(svg_pt);
+    final VectorInstructions instructionsEx = parseWithoutOptimizers(svg_ex);
+    final VectorInstructions instructionsEm = parseWithoutOptimizers(svg_em);
+    final VectorInstructions instructionsRem = parseWithoutOptimizers(svg_rem);
+
+    expect(instructionsPx.paints, <Paint>[
+      const Paint(
+          stroke: Stroke(color: Color(0xff0000ff), width: 1.0),
+          fill: Fill(color: Color(0xffff0000))),
+    ]);
+
+    expect(instructionsPt.paints, <Paint>[
+      const Paint(
+          stroke: Stroke(color: Color(0xff0000ff), width: 1 + 1 / 3),
```

Comment:
Where is the 1/3 magic number coming from? A named constant might be helpful here.

Reply:
from the following [documentation](https://www.w3.org/TR/css3-values/#absolute-lengths)
it is stated that `1pt` is `1/72 of 1in`, and the value of `1in = 96` so the formula is `1/72*96` or `96/72` or `4/3` or `1 + 1/3`. I think i'll change the test to  `pt * 96/72`